scripts/startup.sh

```diff
@@ -62,13 +62,13 @@ if [[ -z $BUCKET ]]; then
 	echo "ERROR: No bucket path detected."
 	exit 1
 fi
-STORAGE_CMD="CLOUDSDK_CORE_UNIVERSE_DOMAIN=$UNIVERSE_DOMAIN gcloud storage"
+
 SCRIPTS_ZIP="$HOME/slurm-gcp-scripts.zip"
-eval $(bash -c "$STORAGE_CMD cp $BUCKET/slurm-gcp-devel.zip $SCRIPTS_ZIP")
-if ! [[ -f "$SCRIPTS_ZIP" ]]; then
-	echo "ERROR: Could not download SlurmGCP scripts."
-	exit 1
-fi
+export CLOUDSDK_CORE_UNIVERSE_DOMAIN="$UNIVERSE_DOMAIN"
```

Comment:
`CLOUDSDK_CORE_UNIVERSE_DOMAIN` where is this used ?

Reply:
in `gcloud storage`